Greeting Card Now Displays Only Once Per Session

Shragupt opened the pull request:
Fixes: #46 

Description:
. The greeting card will now appear only once when the user first opens the site.
. Navigating between pages or reloading the site will not trigger the greeting again in the same session.
. Used sessionStorage to ensure that the greeting is shown only once per session.
. Once the browser tab is closed, the greeting will reset and show again in a new session.

 * Tested and Working:
 . Greeting appears on the first visit.
 . Does not appear again when navigating between pages.
 . Appears again only when the user closes and reopens the tab.

After Implementation:


Uploading Coding Club - Personal - Microsoft_ Edge 2025-02-15 16-58-28.mp4…

ak-0283 commented:
update your branch @Shragupt

Shragupt commented:
I've updated my branch, you can review and merge now @ak-0283

ak-0283 commented:
lack of logo.please check it once. @Shragupt 

![Screenshot 2025-02-16 104430](https://github.com/user-attachments/assets/2d31b64b-10a4-4fbe-9431-3e1d3df7cf27)

Shragupt commented:
I’ve updated my branch, including the greeting fix & blinking effect. Everything is clean and synced. Let me know if anything else needs attention. Everything is working fine on my end. However, I’m not sure why it's not showing as updated on your side. Can you please check once?

ak-0283 commented:
update once more. @Shragupt

Shragupt commented:
I've updated my branch and implemented the required changes @ak-0283 , still if it's not showing then i don't know where the issue is.
Please review and merge.

ak-0283 commented:
i am closing this pr create a new and raise the pr,bcz lots of pages are not updated.  i hope you got my point @Shragupt
